common/src/main/java/com/microsoft/identity/common/internal/platform/AndroidPlatformUtil.java

```diff
@@ -215,7 +215,7 @@ private boolean isValidHubRedirectURIForNAATests(String redirectUri) {
         // Below redirectURI is being used in our automation tests and also by OneAuth tests for NAA
         return BuildConfig.DEBUG && (redirectUri.equals("msauth://com.microsoft.teams/VCpKgbYCXucoq1mZ4BZPsh5taNE=")
                         || redirectUri.equals("msauth://com.microsoft.teams/fcg80qvoM1YMKJZibjBwQcDfOno=")
-                        || redirectUri.equals("msauth://com.microsoft.teams/fcg80qvoM1YMKJZibjBwQcDfOno%3D"));
```

Comment:
I see the description says you are adding a new redirect uri. Is this old one being deleted no longer in use?

Reply:
This one seems the same as the one on the previous line

Reply:
yes, this one was duplicated. I have replaced it with USGov redirectURI